packages/botonic-eslint-config/index.js

```diff
@@ -0,0 +1,135 @@
+// Default eslint configuration
+
+module.exports = {
+  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
+  extends: [
+    'plugin:prettier/recommended', // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    'eslint:recommended',
+    'plugin:jest/recommended',
+    'plugin:node/recommended',
+    'plugin:import/recommended',
+    'plugin:promise/recommended',
+    // typescript
+    'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+    'plugin:@typescript-eslint/eslint-recommended',
+    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+  ],
+  plugins: [
+    '@typescript-eslint',
+    'filenames',
+    'import',
+    'jest',
+    'no-null',
+    'promise',
+    'simple-import-sort',
+  ],
+  parserOptions: {
+    ecmaVersion: 2017, // async is from ecma2017. Supported in node >=7.10
+    sourceType: 'module', // Allows for the use of imports
+    ecmaFeatures: {
+      jsx: true, // Allows for the parsing of JSX
+    },
+  },
+  settings: {
+    'import/parsers': {
+      '@typescript-eslint/parser': ['.ts', '.tsx', '.d.ts'],
+    },
+    'import/resolver': {
+      typescript: {
+        alwaysTryTypes: true, // always try to resolve types under `<root>@types` directory even it doesn't contain any source code, like `@types/unist`
+      },
+    },
+  },
+  // npm run lint runs eslint with --quiet --fix so that only errors are fixed
+  rules: {
```

Comment:
Are the rules based on what you already had on Botonic or where are they coming from?

Reply:
yes, in botonic plugin contentful & dynamo